frontend/src/components/pages/teacher/DisplayAssessmentResultsPage/index.tsx

```diff
@@ -14,12 +15,16 @@ import NotFound from "../../NotFound";
 import DisplayAssessmentResultsByStudentPage from "./DisplayAssessmentResultsByStudentPage";
 import DisplayAssessmentResultsSummaryPage from "./DisplayAssessmentResultsSummaryPage";
 
-const TAB_CONFIG = [
-  {
-    name: "Summary",
-    path: Routes.DISPLAY_ASSESSMENT_RESULTS_SUMMARY_PAGE(),
-    Component: DisplayAssessmentResultsSummaryPage,
-  },
+const makeTabConfig = (enableTeacherDataViz: boolean) => [
```

Comment:
nit: (you don't have to change this one if you don't want to)
```suggestion
const getTabConfig = (enableTeacherDataViz: boolean) => [
```

(I don't love the word `make` as a function prefix)